src/types/sequence.rs

```diff
@@ -282,6 +325,22 @@ where
     Ok(v)
 }
 
+fn extract_sequence_into_slice<'s, T>(obj: &'s PyAny, slice: &mut [T]) -> PyResult<()>
```

Comment:
How about `extract_sequence_inplace`?

Reply:
I tend to think `_inplace` as referring to the first argument, so I would vote against that name.

`extract_sequence_into_slice` is fine imo.